packages/clay-management-toolbar/src/ClayManagementToolbar.js

```diff
@@ -220,6 +229,15 @@ ClayManagementToolbar.STATE = {
 	 */
 	id: Config.string(),
 
+	/**
+	 * Flag to indicate if the Info button is needed.
+	 * @instance
+	 * @memberof ClayManagementToolbar
+	 * @type {?bool}
+	 * @default false
+	 */
+	infoButton: Config.bool().value(false),
```

Comment:
For consistency with the rest of the params I think this should be named `showInfoButton`

Reply:
Oh yeah, it looks so much better now. Thanks!